app/server/lib/DocApi.ts

```diff
@@ -534,7 +534,7 @@ export class DocWorkerApi {
     }));
 
     // Returns the status of any current / pending attachment transfers
-    this._app.get('/api/docs/:docId/attachments/transferStatus', isOwner, withDoc(async (activeDoc, req, res) => {
+    this._app.get('/api/docs/:docId/attachments/transferStatus', canView, withDoc(async (activeDoc, req, res) => {
```

Comment:
Should /attachments/store also be canView for consistency?

Reply:
That's a little leakier, as it reveals (in the future) user label and maybe store type to unauthenticated users. 

Happy with either, I'm not sure if that's a security concern or not.

Reply:
That said - all that info is in the downloaded Grist doc, thinking about it, so it's not private info. So yes, we could make it canView.